app/scripts/services/error.js

```diff
@@ -6,6 +6,8 @@ angular.module('confRegistrationWebApp')
       // Generate a promise catch handler that generates an Error object from an HTTP response object
       errorFromResponse: function (defaultErrorMessage) {
         return function (res) {
+          console.error(res);
```

Comment:
@OzzieOrca Hey Scotty,  was just following your last post.  I agree with you that giving validation feedback next to each input (where possible, or maybe in the same pop up modal) is better UX.  I chimed in because I'm actually working on an Jira issue for ERT (EVENT-489) where there is supposed to be an error popup in the browser that isn't popping up because getting a 500 error from server.  Also made not in code of where in code it is specifically failing. Attached is picture.  Are these the modal with errors that you are referring too?  Because for this particular issue I think having a validation message within the paymentModal.js window would be much better than the notify error message (see attached) that is failing.
![notify_error_message](https://cloud.githubusercontent.com/assets/17660794/22353189/4fce59ce-e3ed-11e6-98e0-5f15b249e69a.png)